Baby Monitor/Source Files/AppDelegate.swift

```diff
@@ -13,12 +13,21 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     
     var window: UIWindow?
     var rootCoordinator: RootCoordinatorProtocol?
-    let appDependencies = AppDependencies()
+    private(set) var appDependencies = AppDependencies()
     
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         window = UIWindow(frame: UIScreen.main.bounds)
         rootCoordinator = RootCoordinator(window!, appDependencies: appDependencies)
         rootCoordinator?.start()
+        rootCoordinator?.onResetApp = { [weak self] in
```

Comment:
I don't think this is needed. `ApplicationResetter` has `localResetCompletionObservable` we can subscribe to and get notified when app is done resetting.

Reply:
Totally! 👍